### PR TITLE
feat(tags): add print-return tag

### PR DESCRIPTION
**Pull request by sid-maddy**
Since we already create the tag embed from the `embed` object in the metadata, we already have the
support to add images in embeds, albeit a bit more verbose than if we had added a `media` property
in the metadata containing only the URL.

Closes https://github.com/python-discord/bot/issues/2222
Closes https://github.com/python-discord/meta/issues/187

### CODE REVIEW

File: bot/resources/tags/print-return.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+embed:
+    image:
+        url: https://cdn.discordapp.com/attachments/267659945086812160/998198889154879558/print-return.gif
+---
+**Print and Return**
```
Comment (Numerlor) [inline bot/resources/tags/print-return.md:6]:
```suggestion
---
embed:
    title: "Print and Return"
    image:
        url: https://cdn.discordapp.com/attachments/267659945086812160/998198889154879558/print-return.gif
---
```
I think a proper title looks better; it's a bit more distinct and doesn't add the big spacing https://i.imgur.com/BRR78id.png

**Review by Numerlor — COMMENTED** [review]
Should we be using discord's cdn for the gif instead of hosting it somewhere ourselves?

**Comment by sid-maddy** [comment]
> Should we be using discord's cdn for the gif instead of hosting it somewhere ourselves?

Do we have our own CDN to host images?
If not, is there any benefit to hosting on a different host?

**Comment by ChrisLovering** [comment]
You can upload the GIF to this repo and then reference the github raw URL for it in the embed.

The benefit is that this embed won't stop working if the original message is delete from within Discord.

**Review by shenanigansd — APPROVED** [review]
LGTM

**Review by lemonsaurus — APPROVED** [review]
Please can we just change all our tags into animated gifs? Holy shit this is so cool. I wish I could approve this more than once. ✅ ✅✅✅✅✅✅✅✅✅

**Pull request merged**